systemd/ceph-disk@.service

```diff
@@ -7,5 +7,7 @@ Wants=local-fs.target
 Type=oneshot
 KillMode=none
 Environment=CEPH_DISK_TIMEOUT=300
+ExecStartPre=-/bin/sh -c "if [[ $(ceph-conf ms_type) = *rdma* ]] || [[ $(ceph-conf ms_public_type) = *rdma* ]] || [[ $(ceph-conf ms_cluster_type) = *rdma* ]] ; then /usr/lib/ceph/systemd-rdma-settings.sh %n; fi"
```

Comment:
`[[` is a bashism, not OK here because `/bin/sh` might be "dash" - see e.g. http://mywiki.wooledge.org/Bashism